### PR TITLE
Add banner

### LINKED ISSUE #1281: Announcement banner
Dev Plan

- Banner, attached to navbar
- Add typed object to feature flags, providing a string route and string for either english text or a key into the localization file
- UTC expiry date (in UTC ISO format), so we don't have to manually remove from config to remove message

### PR DESCRIPTION
**Pull request by jonator**
Closes #1281

### CODE REVIEW

**Comment by sunnya97** [comment]
Task linked: [CU-862j31xqv Banner - FE Dev](https://app.clickup.com/t/862j31xqv)

File: packages/web/components/navbar/index.tsx [hunk 1]
```diff
@@ -239,3 +261,62 @@ const WalletInfo: FunctionComponent<
...
+        "fixed top-[72px] z-[51] float-right my-auto ml-sidebar flex w-[calc(100vw_-_12.875rem)] items-center px-8 py-[14px] md:ml-0 md:w-full",
+        {
+          "bg-gradient-negative": isWarning,
+          "bg-osmoverse-700": !isWarning,
+        }
+      )}
+    >
+      <div className="flex w-full place-content-center items-center gap-3 text-center text-subtitle1">
+        {t(enTextOrLocalizationPath)}{" "}
+        {link && (
```
Comment (JoseRFelix) [inline packages/web/components/navbar/index.tsx:288]:
```suggestion
        {Boolean(link) && (
```

File: packages/web/components/navbar/index.tsx [hunk 2]
```diff
@@ -239,3 +261,62 @@ const WalletInfo: FunctionComponent<
...
+              target="_blank"
+            >
+              {t(link?.enTextOrLocalizationKey ?? "Click here to learn more")}
+            </a>
+            <Image
+              alt="visit link"
+              src="/icons/arrow-right-small.svg"
+              height={24}
+              width={24}
+            />
```
Comment (JoseRFelix) [inline packages/web/components/navbar/index.tsx:303]: We could use the Icon component here

File: packages/web/components/navbar/index.tsx [hunk 3]
```diff
@@ -239,3 +261,62 @@ const WalletInfo: FunctionComponent<
...
+        <button
+          className="flex cursor-pointer items-center"
+          onClick={closeBanner}
+        >
+          <Image
+            alt="close banner"
+            src="/icons/close-small.svg"
+            height={24}
+            width={24}
+          />
```
Comment (JoseRFelix) [inline packages/web/components/navbar/index.tsx:317]: Same as above

File: packages/web/public/icons/arrow-right-small.svg [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<svg width="25" height="24" viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M3 12H21.5" stroke="white" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M14.5 5L21.5 12L14.5 19" stroke="white" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
```
Comment (JoseRFelix) [inline packages/web/public/icons/arrow-right-small.svg:4]: We should add this to the sprites.svg

File: packages/web/public/icons/close-small.svg [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M18 6L6 18" stroke="#E4E1FB" stroke-width="3" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M6 6L18 18" stroke="#E4E1FB" stroke-width="3" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
```
Comment (JoseRFelix) [inline packages/web/public/icons/close-small.svg:4]: Same as above

**Review by JoseRFelix — CHANGES_REQUESTED** [review]
Looks good! left some minor comments

File: packages/web/components/navbar/index.tsx [hunk 6]
```diff
@@ -239,3 +261,52 @@ const WalletInfo: FunctionComponent<
...
+          </div>
+        )}
+      </div>
+      {!persistent && !isWarning && (
+        <button
+          className="flex cursor-pointer items-center"
+          onClick={closeBanner}
+        >
+          <Icon id="close-small" height={24} width={24} />
+        </button>
```
Comment (JoseRFelix) [inline packages/web/components/navbar/index.tsx:308]: This is minor, but I just noticed that we could use the `IconButton` component

**Review by JoseRFelix — APPROVED** [review]

**Pull request merged**